src/packages/cli/src/__tests__/generate.test.ts

```diff
@@ -14,7 +14,7 @@ it('should work with a custom output dir', async () => {
   const { main } = await import(ctx.fs.path('main.ts'))
   expect(cleanSnapshot(data.stdout)).toMatchSnapshot()
   await expect(main()).resolves.toMatchSnapshot()
-})
+}, 10000)
```

Comment:
generate should be way faster than that less than a second on most all machines so maybe changing it like this:
```suggestion
}, 1000)
```

Reply:
![image](https://user-images.githubusercontent.com/284476/92939897-5f19a800-f41c-11ea-953b-4b9828c5a76d.png)

hmm

Reply:
Oh sorry then....